index.html

```diff
@@ -173,7 +173,7 @@ <h3>
             <ol>
               <li>Let |allPorts:list&lt;SerialPort&gt;| be an empty [=list=].
               </li>
-              <li>[=list/For each=] paired Bluetooth device:
+              <li>[=list/For each=] Bluetooth device registered with the system:
```

Comment:
Was not using the "available" definition here intentional? This loop technically creates multiple "available Bluetooth serial ports" so saying "available Bluetooth device" is an abuse of the definition.